### ISSUE TITLE
Fix zoomed in focus highlight

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#144]
Since removing the arrows in https://github.com/phetsims/friction/issues/142, the focusHighlight has changed to be closer wrapping the atoms, and doesn't include the background as much. I should fix that.

**Comment by zepumph** [comment]
Here is a patch where I have tried to get this to work, but found that the way of duplicating the wrappedNode's focusHighlight onto the grabButton isn't general. Right now I we were just setting the centers to each other. @jonathanolson recommended using instead something like 

`wrappedNode.focusHighlight.shape.transformed( wrappedNode.transform.getInverse() )` but that didn't work immediately. I will keep poking around it soon.

<details>
Index: scenery-phet/js/accessibility/nodes/GrabButtonNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- scenery-phet/js/accessibility/nodes/GrabButtonNode.js	(revision c1515f3c8c4a776a1997e922566b4e4639456443)
+++ scenery-phet/js/accessibility/nodes/GrabButtonNode.js	(date 1541554455088)
@@ -9,6 +9,7 @@
   // modules
   const FocusHighlightFromNode = require( 'SCENERY/accessibility/FocusHighlightFromNode' );
   const FocusHighlightPath = require( 'SCENERY/accessibility/FocusHighlightPath' );
+  const GrabReleaseCueNode = require( 'SCENERY_PHET/accessibility/nodes/GrabReleaseCueNode' );
   const KeyboardUtil = require( 'SCENERY/accessibility/KeyboardUtil' );
   const Node = require( 'SCENERY/nodes/Node' );
   const sceneryPhet = require( 'SCENERY_PHET/sceneryPhet' );
@@ -16,7 +17,6 @@
   const StringUtils = require( 'PHETCOMMON/util/StringUtils' );
   const Tandem = require( 'TANDEM/Tandem' );
   const utteranceQueue = require( 'SCENERY_PHET/accessibility/utteranceQueue' );
-  const GrabReleaseCueNode = require( 'SCENERY_PHET/accessibility/nodes/GrabReleaseCueNode' );
 
   // a11y strings
   const grabPatternString = SceneryPhetA11yStrings.grabPattern.value;
@@ -122,8 +122,7 @@
       wrappedNode.focusHighlight = wrappedNodeFocusHighlight;
 
       // Make the grab button's focusHighlight in the spitting image of the wrappedNode's
-      const grabButtonFocusHighlight = new FocusHighlightPath( wrappedNodeFocusHighlight.shape );
-      grabButtonFocusHighlight.center = wrappedNode.center;
+      const grabButtonFocusHighlight = new FocusHighlightPath( wrappedNode.focusHighlight.shape.transformed( wrappedNode.transform.getInverse() ));
       grabButton.focusHighlight = grabButtonFocusHighlight;
 
       // if ever we update the wrappedNode's focusHighlight, then update the grab button's too to keep in syn.
Index: friction/js/friction/view/magnifier/MagnifierNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- friction/js/friction/view/magnifier/MagnifierNode.js	(revision 8c53f467f3755de7d2628af17656ebaa25283c9f)
+++ friction/js/friction/view/magnifier/MagnifierNode.js	(date 1541551633163)
@@ -161,14 +161,8 @@
 
     dragArea.addInputListener( new FrictionDragHandler( model, tandem.createTandem( 'dragAreaDragHandler' ) ) );
 
-    // a11y - The focusHighlight of the top atoms. It also includes the place for the arrows so that it extends up into
-    // the book "background." Dilated to get around the arrows fully. See `atomRowsToEvaporateProperty.link()` below
-    let arrowAndTopAtomsForFocusHighlight = new Node();
-    arrowAndTopAtomsForFocusHighlight.children = [ dragArea ]; // TODO: this is overly complicated, fix this
-
     // a11y - custom shape for the focus highlight, shape will change with atomRowsToEvaporateProperty
-    let focusHighlightShape = Shape.bounds( dragArea.bounds );
-    let focusHighlightPath = new FocusHighlightPath( focusHighlightShape );
+    let focusHighlightPath = new FocusHighlightPath( getFocusHighlightShape( dragArea ) );
     dragArea.setFocusHighlight( focusHighlightPath );
 
     // a11y - add the keyboard drag listener to the top atoms
@@ -275,7 +269,7 @@
       dragArea.setRectHeight( ( number + 2 ) * FrictionModel.MAGNIFIED_ATOMS_INFO.distanceY );
 
       // Update the size of the focus highlight accordingly
-      focusHighlightPath.setShape( Shape.bounds( arrowAndTopAtomsForFocusHighlight.bounds ) );
+      focusHighlightPath.setShape( getFocusHighlightShape( dragArea ) );
     } );
 
     // @private
@@ -298,6 +292,15 @@
     }
   }
 
+  /**
+   *
+   * @param {Node} dragArea
+   * @returns {Shape}
+   */
+  function getFocusHighlightShape( dragArea ) {
+    return Shape.bounds( dragArea.bounds.withOffsets( 0, 40, 0, 0 ) );
+  }
+
   return inherit( Node, MagnifierNode, {
 
     /**
Index: scenery/js/accessibility/FocusHighlightFromNode.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- scenery/js/accessibility/FocusHighlightFromNode.js	(revision cd3fd188cb94ed01168381cd2af860ddd251a6ab)
+++ scenery/js/accessibility/FocusHighlightFromNode.js	(date 1541551948091)
@@ -2,9 +2,9 @@
 
 /**
  * A FocusHighlightPath subtype that is based around a Node. The focusHighlight is constructed based on the bounds of
- * the node. Handles transformations so that when the source node is transformed, the FocusHighlightFromNode will 
+ * the node. Handles transformations so that when the source node is transformed, the FocusHighlightFromNode will
  * updated be as well.
- * 
+ *
  * @author Michael Kauzmann (PhET Interactive Simulations)
  * @author Jesse Greenberg (PhET Interactive Simulations)
  */
@@ -21,6 +21,7 @@
    *
    * @param {Node|null} node
    * @param {Object} [options]
+   * @extends FocusHighlightPath
    * @constructor
    */
   function FocusHighlightFromNode( node, options ) {
</details>

**Comment by zepumph** [comment]
This should be fixed, the rest will be handled in https://github.com/phetsims/scenery-phet/issues/421
